client/src/components/ServiceCard.tsx

```diff
@@ -0,0 +1,55 @@
+import { ReactElement } from 'react';
+import { AiOutlineShareAlt, AiFillStar } from 'react-icons/ai'
+import Service from '../Service';
+
+type Props = {
+    service: Service;
+}
+
+// logic for defining tags is not determined yet
+var tags: string[] = ["a", "b", "c"];
+
+const ServiceCard = ({ service }: Props): ReactElement => {
+    return (
+        <div>
+            <div className="card">
+                <div className="title"> 
+                    <div className="name">
+                        <h6><b>{service.serviceName}</b></h6>
+                        <AiFillStar className="favourite" onClick={()=>{}}/>        
+                    </div>
+                    <div className="ratingItem">
+                        <h6><b>{'5'} / 5</b></h6>
```

Comment:
I hardcoded this placeholder since ratings and reviews are not part of the backend data structure at the moment. Shouldn't be that hard to change it after tho